samples/README.DockerHub.md

```diff
@@ -0,0 +1,30 @@
+# Latest Version of Common Tags
+
+The following tags are the latest stable versions of the most commonly used images. The complete set of tags is listed further down.
+
+- [`4.7.1`](https://github.com/Microsoft/dotnet-framework-docker/blob/master/4.7.1-windowsservercore-1709/runtime/Dockerfile)
```

Comment:
I assume we will want to push both the dotnetapp as well as the aspnetapp.  Should the tags be `dotnetapp-3.5` and `dotnetapp-4.7.2` and mirrored for aspnetapp?